src/utils.jl

```diff
@@ -0,0 +1,33 @@
+@inline sqr(x) = x*x
+@inline cub(x) = x*x*x
+
+"""
+Build `rowspec`, `valspec`, `colspec` such that the product
+
+`out = rowspec * valspec * colspec` will be equivalent to:
+
+```julia
+out = zeros(n, n)
+
+for (i, j, v) in args
+    out[i, j] = v
+end
+```
+
+"""
+function _build_woodbury_specs{T}(::Type{T}, n::Int, args::Tuple{Int, Int, Any}...)
```

Comment:
This has been merged into Woodbury is now (unexported) `WoodburyMatrices. sparse_factors` in the latest tagged release